worlds/mlss/__init__.py

```diff
@@ -55,29 +55,28 @@ class MLSSWorld(World):
     settings: typing.ClassVar[MLSSSettings]
     item_name_to_id = {name: data.code for name, data in item_table.items()}
     location_name_to_id = {loc_data.name: loc_data.id for loc_data in all_locations}
-    required_client_version = (0, 4, 5)
+    required_client_version = (0, 5, 0)
 
-    disabled_locations: List[str]
+    disabled_locations: Set[str] = set()
```

Comment:
Ah, I found the problem. This `set()` will be **shared across all instances of MLSSWorld**.

To make an actual instance attribute per world, you need to also do

`self.disabled_locations = set()` at the start of `generate_early`

Let me see if I can make this suggestion actually

Reply:
Good to know